public/app/features/explore/extensions/getExploreExtensionConfigs.test.tsx

```diff
@@ -23,6 +23,15 @@ describe('getExploreExtensionConfigs', () => {
           onClick: expect.any(Function),
           category: 'Dashboards',
         },
+        {
+          type: 'link',
+          title: 'Add Correlation',
```

Comment:
we generally use sentence case in the UI
```suggestion
          title: 'Add correlation',
```